content/changelog/2025-05-16.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Neon Data API, Neon credits, collapsible console sidebar, and more
+---
+
+## Introducing the Neon Data API (Private Preview)
+
+We're excited to announce the **Neon Data API**, now available in Private Preview! Instantly turn your Neon Postgres database into a REST API. No backend required. Query tables, views, and functions right from your client app using standard HTTP verbs (`GET`, `POST`, `PATCH`, `DELETE`), powered by [PostgREST](https://postgrest.org).
+
+![Data API enabled view with Project URL](/docs/relnotes/data-api-enabled.png)
+
+**What can you do with the Neon Data API?**
+
+- Query your database from any client using HTTP or PostgREST-compatible SDKs (`postgrest-js`, `postgrest-py`, `postgrest-go`)
+- Secure your API with Neon Auth or your own JWKS
+
+Once enabled, you'll get a unique API endpoint for your project. Here's how you might query your data from JavaScript:
+
+```javascript shouldWrap
```

Comment:
I really think this example here should be postgrest-js...

```
      const { data, error } = await postgrest
        .from("notes")
        .select("id, title, created_at, owner_id, shared")
        .eq("owner_id", user.id)
        .order("created_at", { ascending: false });
```

Reply:
fixed.